src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -31,26 +32,49 @@ public abstract class RemoteAuthenticationHandler<TOptions> : AuthenticationHand
 
         protected virtual async Task<bool> HandleRemoteCallbackAsync()
         {
-            var authResult = await HandleRemoteAuthenticateAsync();
-            if (authResult != null && authResult.Skipped)
+            AuthenticateResult authResult = null;
```

Comment:
nit: I think you can declare the ticket here instead of the result, and leave the authResult inside of the try catch, and only set the ticket in a final else